src/Microsoft.DotNet.Build.CloudTestTasks/Microsoft.DotNet.Build.CloudTestTasks.csproj

```diff
@@ -29,11 +29,13 @@
     <Compile Include="AzureConnectionStringBuildTask.cs" />
     <Compile Include="AzureBlobLease.cs" />
     <Compile Include="AzureHelper.cs" />
+    <Compile Include="CopyBlobsToLatest.cs" />
```

Comment:
I'm sure I've asked this question before, but why is this assembly named `CloudTestTasks`?  This doesn't have anything to do with `Test`.

Reply:
I have no good answer. Ideally the generic cloud/azure tasks would be split out, IMO.

Reply:
Filed https://github.com/dotnet/buildtools/issues/1878 to track this.